queries/c_sharp/indents.scm

```diff
@@ -0,0 +1,117 @@
+[
+ (property_declaration)
+ ; (field_declaration)
+ (variable_declaration)
+ (accessor_declaration)
+
+ ; (declaration_list)
+
+ (parameter_list)
+ (type_parameter_list)
+ (accessor_list)
+ (attribute_argument_list)
+ (base_list)
+ (argument_list)
+
+ (binary_expression)
+ (expression_statement)
+ (return_statement)
+ (local_declaration_statement)
+ (local_function_statement)
+ (yield_statement)
+ (fixed_statement)
+ (unsafe_statement)
+ (labeled_statement)
+ (for_each_statement)
+ (for_statement)
+ (if_statement)
+ (switch_expression)
+ (switch_section)
+ (switch_statement)
+ (while_statement)
+ (lambda_expression)
+ (do_statement)
+ (checked_statement)
+ (try_statement)
+ (using_statement)
+ (local_function_statement)
+ (object_creation_expression)
+ (array_creation_expression)
+ (implicit_object_creation_expression)
```

Comment:
Imo, Some of these, from looking at the AST, can be replaced with a single `(block)` capture,

Reply:
I did the whole thing incrementally over time, so i don't remember exactly all the reasons for everything. I did have the idea to use `block` captures, but it didn't work for some reason.

Reply:
It is because of indenting inside of those statements, like:
```csharp
if (
     with()
     && many()
     && conditions()
)
{
    foo();
}
```
which seems to be broken also, as now my query will extra indent at `&&` lines.

Reply:
Ok, so another language like dart, swift and friends where the conditions of the if statement isn't a node :frowning_face: 

Consider this done